client/components/Run/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Scenario from '@components/Scenario';
+
+class Run extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            runId: undefined,
+            scenarioId:
+                this.props.scenarioId || this.props.match.params.scenarioId
+        };
+
+        this.onClickRunScenario = this.onClickRunScenario.bind(this);
+    }
+
+    async componentDidMount() {
+        const runData = await (await fetch(
+            `/api/runs/create/scenario/${this.state.scenarioId}`,
+            {
+                method: 'PUT',
+                headers: {
+                    'Content-Type': 'application/json'
+                }
+            }
+        )).json();
+        if (runData.status === 200) {
+            this.setState({ runId: runData.id });
+        }
+    }
+
+    onClickRunScenario() {
```

Comment:
Is this function being used anywhere? Let me know if I'm missing it.

Reply:
Good catch!